cmd/podman/common.go

```diff
@@ -14,6 +14,14 @@ import (
 	"github.com/urfave/cli"
 )
 
+var (
+	stores     = make(map[storage.Store]struct{})
+	LatestFlag = cli.BoolFlag{
```

Comment:
Don't change now, but I'm just wondering if this should default to True with BoolTFlag.  It's a usability question and I could see it going either way.  Something to ponder.

Reply:
Although if we're using createTime to signify latest rather than modifiedTime, probably best not as default.